packages/web/src/scss/tools/_form-fields.scss

```diff
@@ -0,0 +1,83 @@
+@use '../theme/form-fields' as form-fields-theme;
+
+@use 'accessibility';
+@use 'typography';
+
+@mixin root($type) {
+    @if $type == 'radiofield' or $type == 'checkboxfield' {
```

Comment:
I would prefer not to use specific component name but more common (boolean) attribute.
Maybe like `isBlock`?

Reply:
Maybe box fields and inline fields? At least that is the best I could [come up with in RUI](https://github.com/react-ui-org/react-ui/tree/master/src/lib/styles/tools/form-fields).